docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -39,6 +39,18 @@ Read more in <<uptime-set-up-choose-agent>>.
 
 image:images/uptime-set-up-app.png[]
 
+| **Project monitors** | beta[] With project monitors, you write tests in an external Node.js project and use `@elastic/synthetics` library’s push command to create monitors in {kib}.
```

Comment:
We now support pushing lightweight monitors as of 8.5!

```suggestion

| **Project monitors** | beta[] With project monitors, you write tests in an external version controlled project using Javascript for Browser monitors and YAML for Lightweight monitors, using the `@elastic/synthetics` NPM library’s push command to create monitors in {kib}.
```

I think this sounds a little less scary to boot.

Reply:
Ok... you just snuck this one in!! 🙃  I'll merge as soon as all checks pass.